lib/tasks/journeys_report.rake

```diff
@@ -44,18 +44,23 @@ namespace :journeys do
         moves << {
           supplier: supplier.key,
           reference: move.reference,
+          request_date: move.created_at,
```

Comment:
I'd label this as `created_date`, not `requested_date` to avoid confusion

Reply:
Good point. Re-labeled.